tests/ssl-opt.sh

```diff
@@ -1152,6 +1152,16 @@ run_test() {
         requires_config_enabled MBEDTLS_SSL_PROTO_DTLS
     fi
 
+    # Check more TLS protocol features.
+    case "$SRV_CMD $CLI_CMD" in
+        *[-_\ =]tickets=[^0]*)
```

Comment:
This is probably correct, but out of curiosity: why allow `=`? Perhaps a comment with examples of things we're meaning to match would help. (I'm not even sure why we allow `-`: is it for the sake of OpenSSL/GnuTLS? But then they don't mandate the `=` sign between the option name and its value, do they?)

Reply:
I used the same punctuation before the word in several places, including `PSK` which could come at the beginning of a cipher suite. It happens that `tickets` is only relevant for mbedtls: openssl and gnutls only have `-no_ticket` and `--noticket` respectively. So I could change this one to require a space before. I'm not sure whether to go for uniformity or to get each case exactly right on its own. Getting each case exactly right isn't necessarily a good thing anyway since it's fragile, and being too picky could lead to tests not being executed when they're supported, which is bad.

Reply:
Ok, let's go for uniformity then, especially now that all these tests are grouped in the same function.